### ISSUE TITLE
0.8.2 New feature network multiplier cause crashed while use multi-GPUs training LoRA(SD1.5).

### ISSUE THREAD

**Issue by fireicewolf** [kohya-ss/sd-scripts#1083]
With Kaggle 2xT4 and PyTorch 2.1.2 environment, train sd1.5 LoRA occured issue belowed.

> 
```
Traceback (most recent call last):
  File "/kaggle/working/kohya-sd-scripts/train_network.py", line 1033, in <module>
    trainer.train(args)
  File "/kaggle/working/kohya-sd-scripts/train_network.py", line 777, in train
    network.set_multiplier(multipliers)
  File "/opt/conda/envs/venv/lib/python3.10/site-packages/torch/nn/modules/module.py", line 1695, in __getattr__
    raise AttributeError(f"'{type(self).__name__}' object has no attribute '{name}'")
AttributeError: 'DistributedDataParallel' object has no attribute 'set_multiplier'
``` 

It looks like the 0.8.2 New feature network multiplier caused this issue, I didn't add this parmeter in my dataset.toml,
but it has been auto added.
```
[Dataset 0]
  batch_size: 8
  resolution: (768, 768)
  enable_bucket: True
  network_multiplier: 1.0
  min_bucket_reso: 512
  max_bucket_reso: 1536
  bucket_reso_steps: 64
  bucket_no_upscale: True
``` 
I checked source code in train_network.py, found it was caused by this part.
https://github.com/kohya-ss/sd-scripts/blob/c576f806397f7bca1085518616f8aaebfa2e98b4/train_network.py#L768-L777

I tried to replace
```python
network.set_multiplier(multipliers)
``` 
to 
```python
accelerator.unwrap_model(network).set_multiplier(multipliers)
``` 
This issue not occured again.

But I‘m not familiar with accelerator usage so I don't know whether this will cause any other issues.

@kohya-ss if you have time, please help to check this, Thanks.

**Comment by fireicewolf** [comment]
> Thank you for reporting this! Your fix seems to be correct. I forgot multiple GPU training. I will fix it sooner.

Fine, I will give this a PR, thanks for your check!

**Comment by fireicewolf** [comment]
Fixed in 0.8.3, this issue closed.
